assets/js/modules/analytics-4/components/settings/AdsConversionIDSettingsNotice.test.js

```diff
@@ -32,7 +32,7 @@ describe( 'AdsConversionIDSettingsNotice', () => {
 		registry = createTestRegistry();
 		registry.dispatch( CORE_USER ).receiveGetDismissedItems( [] );
 
-		now = registry.select( CORE_USER ).getReferenceDate( { parsed: true } );
+		now = registry.select( CORE_USER ).getReferenceDate();
```

Comment:
I think this was renamed to `now` when it was returning a `Date` which is a bit confusing when it is now a string. Let's be consistent and name it by the selector: `referenceDate`.

Reply:
Renamed to `referenceDate`.